pulsar-websocket/src/main/java/org/apache/pulsar/websocket/service/WebSocketProxyConfiguration.java

```diff
@@ -67,9 +67,13 @@
     )
     private String globalZookeeperServers;
 
+    @Deprecated
     @FieldContext(doc = "Connection string of configuration store servers")
     private String configurationStoreServers;
 
+    @FieldContext(doc = "Connection string of configuration metadata store servers")
+    private String configurationMetadataStoreUrl;
```

Comment:
Looks like we should be compatible with the old config, overwrite the `getConfigurationMetadataStoreUrl()`, so like:
```java
    public String getConfigurationMetadataStoreUrl() {
        if (StringUtils.isNotEmpty(this.configurationMetadataStoreUrl)) {
            return this.configurationMetadataStoreUrl;
        }

        // backward compatible
        if (StringUtils.isNotEmpty(this.configurationStoreServers)) {
            return this.configurationStoreServers;
        }
        return this.globalZookeeperServers;
    }
```

Reply:
This has already been done in `ServiceConfiguration.getConfigurationMetadataStoreUrl`. When initializing the `WebSocketService`, we will convert `WebSocketProxyConfiguration` to `ServiceConfiguration` via `PulsarConfigurationLoader.convertFrom`. Backward compatibility has been done in `ServiceConfiguration`. 

> You also need to check the usage, use `WebSocketProxyConfiguration#getConfigurationMetadataStoreUrl()` instead of `globalZookeeperServers()` and `getConfigurationStoreServers()`.

I have already checked that. There are no direct use of `WebSocketProxyConfiguration#getConfigurationMetadataStoreUrl()` and there shouldn't be.